workers/unity/Assets/Playground/Scripts/Cubes/CubeMovementSystem.cs

```diff
@@ -19,20 +20,25 @@ public struct Data
 
         [Inject] private Data data;
 
-        private static Vector3 speed = new Vector3(2, 0, 0);
-
-        private Vector3 origin;
-
-        protected override void OnCreateManager(int capacity)
+        public struct WorkerData
```

Comment:
it doesn't seem like we have a convention for that. After checking a few files we usually make them `public`. Would prefer to leave it like this to have it the same everywhere and then create a new PR to make it private everywhere. Otherwise this PR will just become even messier :P